beancount_n26/__init__.py

```diff
@@ -127,6 +128,7 @@ def __init__(
         self,
         iban: str,
         account: str,
+        transferwise_fees_account: str,
```

Comment:
What do you think about removing "Transferwise" from the name? We could call it simply `exchange_fees_account` or something like that.

Reply:
You're right, 6e12e7fd0fb77c8679b274adc9ffacea80dc785f address that